### PR TITLE
Staging: Stage to prod build (10th Apr)

### PR DESCRIPTION
**Pull request by rohit-walkover**

### CODE REVIEW

File: apps/web-giddh-admin/src/app/services/subscription.service.ts [hunk 1]
```diff
@@ -100,10 +107,37 @@ export class SubscriptionService {
...
     public setGetAllCompanyRequestObject(model: GetAllCompaniesRequest) {
         this.getCompanyfilter = model;
     }
+
     public getGetAllCompanyRequestObject(): GetAllCompaniesRequest {
         return this.getCompanyfilter;
     }
+
+    public updateTransactions(subscriptionId: string, body: any) {
+        let url = SUBSCRIPTION_API.UPDATE_TRANSACTIONS
```
Comment (ravinderwalkover) [inline apps/web-giddh-admin/src/app/services/subscription.service.ts:120]: add semicolon

File: apps/web-giddh-admin/src/app/services/subscription.service.ts [hunk 2]
```diff
@@ -100,10 +107,37 @@ export class SubscriptionService {
...
+            .pipe(
+                map((resp) => {
+                    return resp;
+                }),
+                catchError((e) =>
+                    this.errorHandler.HandleCatch(e)));
+    }
+
+    public assignPlan(body: any) {
+        let url = SUBSCRIPTION_API.ASSIGN_PLAN
```
Comment (ravinderwalkover) [inline apps/web-giddh-admin/src/app/services/subscription.service.ts:133]: add semicolon

File: apps/web-giddh-admin/src/app/subscription/components/suscription-container/subscription-container.component.ts [hunk 3]
```diff
@@ -486,4 +517,50 @@ export class SubscriptionContainerComponent implements OnInit {
...
+    public hideAssignPlanPopup(): void {
+        this.openAssignPlanPopup = false;
+        if (this.isFromAdvanceSearchRes) {
+            this.getAdvancedSearchedSubscriptions(this.advanceSearchRequest);
+        } else {
+            this.getSubscriptionData(this.subscriptionRequest);
+        }
+    }
+
+    public showRadioButton(subscription) {
```
Comment (ravinderwalkover) [inline apps/web-giddh-admin/src/app/subscription/components/suscription-container/subscription-container.component.ts:559]: add doc

File: apps/web-giddh-admin/src/app/subscription/components/suscription-container/subscription-container.component.ts [hunk 4]
```diff
@@ -486,4 +517,50 @@ export class SubscriptionContainerComponent implements OnInit {
...
+        this.visibleSubscriptionRadioButton = subscription.subscriptionId;
+    }
+
+    public hideRadioButton() {
```
Comment (ravinderwalkover) [inline apps/web-giddh-admin/src/app/subscription/components/suscription-container/subscription-container.component.ts:563]: add doc

File: apps/web-giddh-admin/src/app/subscription/components/user-list/user-list.component.ts [hunk 5]
```diff
@@ -81,6 +95,14 @@ export class UserListComponent implements OnInit {
         this.showTaxPopup = isShow ? false : true;
     }
 
+    public ownerFocus(isShow: boolean): void {
+        this.owner = isShow ? false : true;
+    }
+
+    public lastSeenDropdown(isShow: boolean): void {
```
Comment (ravinderwalkover) [inline apps/web-giddh-admin/src/app/subscription/components/user-list/user-list.component.ts:102]: add doc

File: apps/web-giddh-admin/src/app/subscription/components/user-list/user-list.component.ts [hunk 6]
```diff
@@ -81,6 +95,14 @@ export class UserListComponent implements OnInit {
         this.showTaxPopup = isShow ? false : true;
     }
 
+    public ownerFocus(isShow: boolean): void {
```
Comment (ravinderwalkover) [inline apps/web-giddh-admin/src/app/subscription/components/user-list/user-list.component.ts:98]: add doc

File: apps/web-giddh-admin/src/app/subscription/components/suscription-container/subscription-container.component.ts (hunk 4 above)
Comment (rohit-walkover) [reply apps/web-giddh-admin/src/app/subscription/components/suscription-container/subscription-container.component.ts:563]: ravinder ji this is your changes

File: apps/web-giddh-admin/src/app/subscription/components/user-list/user-list.component.ts (hunk 5 above)
Comment (rohit-walkover) [reply apps/web-giddh-admin/src/app/subscription/components/user-list/user-list.component.ts:102]: sadique this is your changes please add

File: apps/web-giddh-admin/src/app/subscription/components/suscription-container/subscription-container.component.ts (hunk 3 above)
Comment (rohit-walkover) [reply apps/web-giddh-admin/src/app/subscription/components/suscription-container/subscription-container.component.ts:559]: ravinder ji this is your changes

File: apps/web-giddh-admin/src/app/services/subscription.service.ts (hunk 2 above)
Comment (rohit-walkover) [reply apps/web-giddh-admin/src/app/services/subscription.service.ts:133]: ravinder ji this is your changes

File: apps/web-giddh-admin/src/app/services/subscription.service.ts (hunk 1 above)
Comment (rohit-walkover) [reply apps/web-giddh-admin/src/app/services/subscription.service.ts:120]: ravinder ji this is your changes

**Comment by rohit-walkover** [comment]
@ravindersinghdang @sadik-walkover Please add your changes ASAP so we can make live

**Pull request merged**